### ISSUE TITLE
GMB API Deprecations Update

### ISSUE THREAD

**Issue by tonycoco** [google/alligator2#13]
**TL;DR: We have updated Alligator to align with the GMB API deprecations happening on Apr. 30. There are some scheme changes in the locations table that might require manual steps in BigQuery and Data Studio.**

Hello Alligator 2.0 community,

As we announced last year, the GMB API is changing to a new federated model, and the migration is happening progressively.

On April 30, locations will be available only through API Business Information v1. With that in mind, we have released a new version of Alligator that uses this new endpoint to retrieve the locations, but this implies a new schema for the location objects coming from the new API endpoint.

In practical terms, the functionality offered by Alligator remains the same, but the Locations table will have a new schema that is not compatible with the previous version, so the customer will need to remove the Locations table before running the new version of Alligator, otherwise it will fail.

If you are using Data Studio (or any other visualization tool), you might need to modify the connectors or views as well (we updated the sample SQL files used to generate the views in BigQuery to reflect the new schema).

The deadline to migrate to the new Alligator version is April 30, after which the old locations endpoint will stop working on the GMB API side.

Thanks!

– The Google Team

**Comment by tonycoco** [comment]
Closed with f6ee96d54abd5140fd6830b4b1d7a712d9816231
